addons/library.kodi.guilib/libKODI_guilib.h

```diff
@@ -825,13 +825,13 @@ class CAddonGUIRenderingControl
   virtual ~CAddonGUIRenderingControl();
   virtual void Init();
 
-  virtual bool Create(int x, int y, int w, int h, void *device);
+  virtual bool Create(int x, int y, int w, int h, float ratio);
```

Comment:
"i" don't as such, code to do so has always been there. https://github.com/FernetMenta/xbmc/blob/master/xbmc/addons/Visualisation.cpp#L65